src/test/ElementPropertyExporter.test.ts

```diff
@@ -1,32 +1,33 @@
-import { Logger, LogLevel } from "@bentley/bentleyjs-core";
-/*---------------------------------------------------------------------------------------------
-* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
-* See LICENSE.md in the project root for license terms and full copyright notice.
-*--------------------------------------------------------------------------------------------*/
-import { BackendLoggerCategory, IModelHost, SnapshotDb } from "@bentley/imodeljs-backend";
-import { ElementPropertyExporter } from "../export/ElementPropertyExporter";
-import { TestUtils } from "./TestUtils";
-
-describe("ElementPropertyExporter", () => {
-  before(async () => {
-    await IModelHost.startup();
-    // optionally initialize logging
-    if (false) {
-      Logger.initializeToConsole();
-      Logger.setLevelDefault(LogLevel.Error);
-      Logger.setLevel(BackendLoggerCategory.IModelExporter, LogLevel.Trace);
-    }
-  });
-
-  after(async () => {
-    await IModelHost.shutdown();
-  });
-
-  it.skip("should export element properties", () => {
-    const outputFileName = TestUtils.initOutputFile("ElementPropertyExporter.json");
-    const iModelFileName = ""; // NOTE: replace with path to snapshot iModel
-    const iModelDb = SnapshotDb.openFile(iModelFileName);
-    ElementPropertyExporter.export(iModelDb, outputFileName);
-    iModelDb.close();
-  });
-});
+import { Logger, LogLevel } from "@itwin/core-bentley";
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+import { IModelHost, SnapshotDb } from "@itwin/core-backend";
+import { TransformerLoggerCategory } from "@itwin/core-transformer";
+import { ElementPropertyExporter } from "../export/ElementPropertyExporter";
+import { TestUtils } from "./TestUtils";
+
+describe("ElementPropertyExporter", () => {
+  before(async () => {
+    await IModelHost.startup();
+    // optionally initialize logging
+    if (false) {
+      Logger.initializeToConsole();
+      Logger.setLevelDefault(LogLevel.Error);
+      Logger.setLevel(TransformerLoggerCategory.IModelExporter, LogLevel.Trace);
+    }
+  });
+
+  after(async () => {
+    await IModelHost.shutdown();
+  });
+
+  it.skip("should export element properties", async () => {
```

Comment:
Does this need to be skipped?

Reply:
I'll rename the test because it requires you to supply your own file. It's more of a bin than a test.

I think a rename of the test + comment about how you can turn it on to run a local export suffices here.